src/fragments/lib/push-notifications/android/getting_started/20_cli_resources.mdx

```diff
@@ -0,0 +1,39 @@
+> Prerequisite: [Install and configure the Amplify CLI](/cli/start/install)
+
+<Callout warning>
+
+Push Notifications requires version **10.8.0+** of the Amplify CLI. You can check your current version by typing `amplify -version` and upgrade to the latest version by typing `amplify upgrade`.
+
+</Callout>
+
+<Callout>
+
+If you want to use existing AWS Pinpoint resources [follow this guide](/lib/push-notifications/existing-resources) instead.
```

Comment:
Following this link results in 404.

Reply:
We should just remove this callout completely since we have the switcher.